tests/e2e/taqueria-plugin-contract-types.spec.ts

```diff
@@ -123,13 +123,13 @@ describe("E2E Testing for taqueria contract types plugin",  () => {
             await exec(`cp e2e/data/timelock.tz ${taqueriaProjectPath}/artifacts`)
             await exec(`cd ${taqueriaProjectPath} && taq generate types`)
             await exec(`cp e2e/data/timelock.ts ${taqueriaProjectPath}/types`)
-            await exec(`cd ${taqueriaProjectPath}/types &&
-                        npm init -y &&
-                        npm install typescript --save-dev &&
-                        npm install @taquito/taquito &&
-                        npm install @taquito/signer &&
-                        npx tsc timelock.ts`)
-            const {stdout, stderr} = await exec(`cd ${taqueriaProjectPath}/types && node timelock.js`)
+            await exec(`cp e2e/data/tsconfig.timelock.json ${taqueriaProjectPath}/types`)
+            await exec(`npm init -y`, {cwd: `./${taqueriaProjectPath}/types`})
+            await exec(`npm install typescript --save-dev`, {cwd: `./${taqueriaProjectPath}/types`})
```

Comment:
@jchenche this looks good to me, the only thing I am wondering is if we can move these dependencies to the tests package.json so that they don't need to be installed during the test and can be used by everything in the test folder. I know the package.json already includes the @taquito/taquito package and typescript. This might cut down the number of commands in this test

Reply:
Done!